contrib/test/integration/critest.yml

```diff
@@ -16,6 +16,13 @@
     enabled: yes
     daemon_reload: yes
 
+- name: Check for journald logs
```

Comment:
Check crio logs

Reply:
Actually we aren't looking at the journal, we are checking the status which gives only a few recent lines of the journal